docs/how_to_guides/create_and_edit_expectations/how_to_use_check_assets.rst

```diff
@@ -1,31 +1,33 @@
-.. _check_assets_reference:
+.. _how_to__use_check_assets:
 
 ############################
-Check Assets Pattern
+Build Check Assets
```

Comment:
"How to use check assets to validate data." 

* Please start with "How to..."
* I'd prefer to not capitalize "check assets", since it's not a formal concept in our lexicon. Alternatively, we can add an article in the Reference section, then capitalize and link it.

Reply:
updated